dynawo/sources/Modeler/ModelManager/DYNModelManagerCommon.h

```diff
@@ -43,13 +43,17 @@
 #include "DYNMessage.hpp"
 #include "DYNMessageTimeline.h"
 #include "DYNTimeline_keys.h"
-#include "DYNModelManagerOwnTypes.h"  ///< redefinition of local own types : should be before simulation_data.h
-#include "simulation_data.h"
 #include "ModelicaStandardTables.h"
 #include "ModelicaStrings.h"
-#include "DYNModelManagerOwnFunctions.h"  ///< redefinition of local own functions
 #include "ModelicaUtilities.h"
 
+// clang-format off
```

Comment:
Since the usage of clang-format has not been yet approved, please remove this

Reply:
OK